src/Microsoft.Extensions.Logging.Console/ConsoleLogScope.cs

```diff
@@ -0,0 +1,89 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+
+#if NET451
+using System.Runtime.Remoting;
+using System.Runtime.Remoting.Messaging;
+#else
+using System.Threading;
+#endif
+
+namespace Microsoft.Extensions.Logging.Console
```

Comment:
I thought about this but left it like this because we allow users to override `WriteMessage` and probably they would be interested in getting the log scope to write the way they want to